app/views/user_mailer/registration_confirmation.html.slim

```diff
@@ -0,0 +1,5 @@
+p Hi #{@user.name},
+
+p Thanks for registering! To confirm your registration click the URL below.
+
+href="http://localhost:3000/sign_up/confirm_email?token=#{@user.confirm_token}"
```

Comment:
Will we change the localhost to the real URL later?

Reply:
I think we'll change it later. Because there is a moment with https on heroku.